django/utils/html.py

```diff
@@ -226,8 +228,7 @@ def unquote_quote(segment):
         # urlencode will take care of quoting
         query = urlencode(query_parts)
 
-    path = unquote_quote(path)
-    fragment = unquote_quote(fragment)
+    # Leave fragment unaltered.
```

Comment:
So the question is whether it's safe to do this. (From security or regressions POV.)